src/infrastructure/repository/postgresql-task-repository.ts

```diff
@@ -34,4 +34,17 @@ export class PostgresqlTaskRepository implements TaskRepositoryInterface {
 
     return new Task(row);
   }
+
+  public async findById(id: string) {
+    const [row] = await this.database
+      .select()
+      .from(tasks)
+      .where(eq(tasks.id, id));
+
+    if (!row) {
+      return undefined;
+    }
+
+    return new Task(row);
+  }
```

Comment:
[ask][imo]

- tasksテーブルのカラムが増えたりしたときに不要な列をselectしちゃうのを防ぐ
- Taskのインスタンス化するときに不要な値渡すのを防ぐ

以上の理由から、このように明示的に書くやりかたもあると思うのですが、どうでしょうか？ :eyes:
なにかご意見あれば聞きたいです

```suggestion
  public async findById(id: string) {
    const [row] = await this.database
      .select({
        id: tasks.id,
        title: tasks.title,
        done: tasks.done,
      })
      .from(tasks)
      .where(eq(tasks.id, id));

    if (!row) {
      return undefined;
    }

    return new Task({
      id: row.id,
      title: row.title,
      done: row.done,
    });
  }
```

Reply:
@gn-t-k 
下記で修正しました🫡
(ついでにsave時もカラム指定して取得するようにしてます)
[8f65c9a](https://github.com/praha-inc/praha-challenge-ddd-template/pull/31/commits/8f65c9ad5bb02dae1f8a5c929ee02ac08fdcd424)